sklearn/ensemble/bagging.py

```diff
@@ -281,7 +283,7 @@ def _fit(self, X, y, max_samples=None, max_depth=None, sample_weight=None):
         random_state = check_random_state(self.random_state)
 
         # Convert data
-        X, y = check_X_y(X, y, ['csr', 'csc'])
+        X, y = check_X_y(X, y, ['csr', 'csc'], multi_output=self.multi_output)
```

Comment:
After second thoughs, i think @jnothman has a point. We can make `BaggingRegressor` support multi-output targets by simply

- Forcing `multi_output=True` in the call `check_X_y(...)`
- Skipping the `_validate_y(...)` thing. I guess the question should we really be skipping this ?